app/src/main/java/ru/woodymsk/socialapp/presentation/registration/RegistrationFragment.kt

```diff
@@ -47,63 +56,60 @@ class RegistrationFragment : Fragment() {
     override fun onViewCreated(view: View, savedInstanceState: Bundle?) {
         super.onViewCreated(view, savedInstanceState)
 
-            observeRegistrationFormState()
-            observeRegistrationResult()
-            setListeners()
+        observeRegistrationEvents()
+        setListeners()
 
-            binding.bRegistrationScreenSignInButton.setOnClickListener { registration() }
+        binding.bRegistrationScreenSignInButton.setOnClickListener { registration() }
     }
 
     private fun showGreetingToast(userName: String) {
         val welcome = getString(welcome)
         Toast.makeText(requireActivity(), "$welcome, $userName.", Toast.LENGTH_LONG).show()
     }
 
-    private fun observeRegistrationFormState() {
-        viewModel.registrationFormState.observe(viewLifecycleOwner) { registrationFormState ->
+    private fun observeRegistrationEvents() {
+        viewModel.registrationEvents.observe(viewLifecycleOwner) { registrationEvents ->
```

Comment:
```suggestion
        viewModel.registrationEvents.observe(viewLifecycleOwner) { registrationEvent ->
```

у тебя ведь один конкретный ивент
а можешь вообще просто `event` назвать, и так понятно, в принципе.

Reply:
Переделал всё на event в авторизации и регистрации